scripts/src/tests/helpers.ts

```diff
@@ -1,17 +1,21 @@
 import { getManager } from "typeorm";
 import * as StellarSdk from "stellar-sdk";
 
-import { generateId } from "../utils/utils";
+import { generateId, readKeysDir, random } from "../utils/utils";
 import { Asset, Offer } from "../models/offers";
 import { User, AuthToken } from "../models/users";
-import { Application, ApplicationConfig } from "../models/applications";
+import { Application, ApplicationConfig, StringMap } from "../models/applications";
 import { LimitConfig } from "../config";
 import { createEarn, createSpend } from "../create_data/offers";
 import { Poll, PageType } from "../public/services/offer_contents";
 import { CompletedPayment, paymentComplete } from "../internal/services";
 import { ExternalOrder, MarketplaceOrder, Order } from "../models/orders";
 import * as payment from "../public/services/payment";
 import { Event } from "../analytics";
+import { getConfig } from "../internal/config";
+import * as path from "path";
```

Comment:
please move all node_modules imports to the top of the file

Reply:
Done